utils/ips/ip_port.go

```diff
@@ -12,7 +12,44 @@ import (
 	"github.com/ava-labs/avalanchego/utils/wrappers"
 )
 
-var errBadIP = errors.New("bad ip format")
+const nullStr = "null"
+
+var (
+	errMissingQuotes = errors.New("first and last characters should be quotes")
+	errBadIP         = errors.New("bad ip format")
+)
+
+type IPDesc (IPPort)
```

Comment:
Why aren't we just using `IPPort`'s json format?

Reply:
`IPPort` would not work for `1.2.3.4:[PORT]` in the flag parsing, but if we deprecate the bootstrap-ips and port flags, just requiring the user to specify separate fields for ip and port makes sense.